app/templates/components/goodcity/designate-link.hbs

```diff
@@ -1,17 +1,5 @@
 <div class="designate-link">
-  <div {{action 'beginDesignation' pkg order}}>
+  <div {{action 'beginDesignation'}}>
     {{yield}}
   </div>
-
-  {{#if readyToDesignate}}
-    {{goodcity/designate-form
-      quantity=designationQty
-      maxQuantity=designatableQuantity
-      editableQty=editableQty
-      confirmAction=(action "completeDesignation")
-      cancelAction=(action "cancelDesignation")
-      order=designationTargetOrder
-      pkg=designationTargetPackage
-    }}
-  {{/if}}
```

Comment:
This component had the whole designation flow, not it only has half. Feels like a downgrade.
With this change, it'll be very hard for someone to use it without knowing the intricate details of how it works.

If this component is not appropriate for use in the page you're building it's probably best not to use it and use a normal button with your custom actions. This feels too damaging

Reply:
Oh nevermind, I just saw the global component. Ballsy move, but high reward if it works out well